### PR TITLE
Banner: Change copy for PC

### PR DESCRIPTION
**Pull request by delfipolito**
image

### CODE REVIEW

File: src/components/Proposals/ProposalBanner.js [hunk 1]
```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
             color: ${theme.contentSecondary};
           `}
         >
-          We believe humankind should use technology as a liberating tool to
-          unleash all the goodwill and creativity of our species, rather than as
-          a tool to enslave and take advantage of one another.
+          This is a test <br /> Aragon Network Cash is a test organization set
```
Comment (andy-hook) [inline src/components/Proposals/ProposalBanner.js:75]: Is this the final campaign copy? it reads a little strange to me in this configuration. Should "This is a test" be in the paragraph? or is it intended as the title?

File: src/components/Proposals/ProposalBanner.js [hunk 2]
```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
...
+          up as part of a new round of precedence campaign disputes. Learn more
+          on the Aragon project blog
```
Comment (andy-hook) [inline src/components/Proposals/ProposalBanner.js:77]:
```suggestion
          up as part of a new round of precedence campaign disputes. Learn more
          on the Aragon project blog.
```

I'm also wondering should we link to the post in this same sentence? e.g:

Learn more on the [Aragon project blog](https://aragon.org/blog).

File: src/components/Proposals/ProposalBanner.js [hunk 3]
```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
...
+            display: flex;
+            flex-wrap: wrap;
+            align-items: center;
+            justify-content: center;
           `}
-        />
+        >
+          <Button
+            label="Learn more"
+            mode="normal"
```
Comment (andy-hook) [inline src/components/Proposals/ProposalBanner.js:89]: As this is the first action in the pair (and more important) it should probably be `mode="strong"` with the manifesto link becoming the secondary.

File: src/components/Proposals/ProposalBanner.js [hunk 4]
```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
...
+            href="https://aragon.org/blog"
```
Comment (andy-hook) [inline src/components/Proposals/ProposalBanner.js:90]: Is there a specific blog post or area we can link to about this campaign?

File: src/components/Proposals/ProposalBanner.js [hunk 5]
```diff
@@ -73,19 +72,40 @@ function ProposalBanner({ onCloseBanner, ...props }) {
...
+            css={`
+              padding: ${2.75 * GU}px ${3.5 * GU}px;
+              font-weight: 600;
+              margin: ${3 * GU}px ${2 * GU}px 0;
```
Comment (andy-hook) [inline src/components/Proposals/ProposalBanner.js:94]: We should move the `margin-top` to the parent and internally space the buttons separately. I also feel the gap between the buttons is a little too generous and could be tighter, what do you think?

**Review by andy-hook — COMMENTED** [review]
Thanks for adding this! I was wondering about the content and whether we have it quite right. "This is a test" looks a bit strange to me like this. @john-light is this all accurate?

File: src/components/Proposals/ProposalBanner.js (hunk 4 above)
Comment (delfipolito) [reply src/components/Proposals/ProposalBanner.js:90]: Yes, but the post is not out yet, so im gonna link it in the last minute to the specific post

File: src/components/Proposals/ProposalBanner.js (hunk 5 above)
Comment (delfipolito) [reply src/components/Proposals/ProposalBanner.js:96]: Margin top is there to separate the  buttons from each others on mobile

**Review by andy-hook — APPROVED** [review]
Tweaked the button spacing a touch, otherwise LGTM, let's check in with @john-light about the blog link, copy question and amend later if needed.

File: src/components/Proposals/ProposalBanner.js (hunk 5 above)
Comment (andy-hook) [reply src/components/Proposals/ProposalBanner.js]: Sounds good, we can adjust this after we have better responsive utils merged.

**Pull request merged**